app/src/main/java/io/apicurio/registry/mt/TenantRequestFilter.java

```diff
@@ -31,14 +34,21 @@
 
     @Inject
     TenantContext tenantContext;
-    
+
+    @Inject
+    @Current
+    TenantMetadataService tenantMetadataService;
+
     /**
      * @see javax.ws.rs.container.ContainerRequestFilter#filter(javax.ws.rs.container.ContainerRequestContext)
      */
     @Override
     public void filter(ContainerRequestContext requestContext) throws IOException {
         String tenantId = requestContext.getHeaderString("X-Registry-Tenant-Id");
         if (tenantId != null) {
+            //TODO use the metadata
```

Comment:
What's the purpose of fetching the metadata here?

Reply:
nothing actually, I used it to test my changes and this request interceptor is the only thing I had

Reply:
Fair enough. Maybe then change this todo to remove metadata usage or something like that.